components/ChipTextField/examples/MDCChipTextFieldScrollView.h

```diff
@@ -20,33 +20,74 @@
 
 @protocol MDCChipTextFieldScrollViewDelegate <NSObject, UIScrollViewDelegate>
 
-// TODO: delegate methods to handle touch event
+@optional
```

Comment:
Same comment as before - one method doesn't need to be optional.

Reply:
Addressed in 0003b1507a139f7d11899e583f895bac70dd94e6